docs/notifications/getting-started.md

```diff
@@ -0,0 +1,168 @@
+  pluginId: 'myPlugin',
+  register(env) {
+    env.registerInit({
+      deps: {
+        // ...
+        notificationService: notificationService,
+      },
+      async init({ config, logger, httpRouter, notificationService }) {
+        httpRouter.use(
+          await createRouter({
+            // ...
+            notificationService,
+          }),
+        );
+      },
+    });
+  },
+});
+```
+
+To emit a new notification:
+
+```ts
+notificationService.send({
+  recipients /* of the broadcast or entity type */,
+  payload /* actual message */,
+});
+```
+
+Refer the [API documentation](https://github.com/backstage/backstage/blob/master/plugins/notifications-node/api-report.md) for further details.
+
+### Signals
+
+The use of signals with notifications is optional but generally enhances user experience and performance.
+
+When a notification is created, a new signal is emitted to a general-purpose message bus to announce it to subscribed listeners.
+
+The frontend maintains a persistent connection (web socket) to receive these announcements from the notifications channel.
+The specific details of the updated or created notification should be retrieved via a request to the notifications API, except for new notifications, where the payload is included in the signal for performance reasons.
+
+In a frontend plugin, to subscribe for notifications' signals:
+
+```ts
+import { useSignal } from '@backstage/plugin-signals-react';
+
+const { lastSignal } = useSignal<NotificationSignal>('notifications');
+
+React.useEffect(() => {
+  /* ... */
+}, [lastSignal, notificationsApi]);
+```
+
+### Consuming Notifications
+
+In a front-end plugin, the simplest way to query a notification is by its ID:
+
+```ts
+import { useApi } from '@backstage/core-plugin-api';
+import { notificationsApiRef } from '@backstage/plugin-notifications';
+
+const notificationsApi = useApi(notificationsApiRef);
+
+notificationsApi.getNotification(yourId);
+
+// or with connection to signals:
+notificationsApi.getNotification(lastSignal.notification_id);
+```
+
+### External Services
+
+When the emitter of a notification is a Backstage backend plugin, it is mandatory to use the integration via `@backstage/plugin-notifications-node` as described above.
+
+If the emitter is a service external to Backstage, an HTTP POST request can be issued directly to the API, assuming that authentication is properly configured.
+Refer to the service-to-service auth documentation for more details, focusing on the Static Tokens section for the simplest setup option.
+
+An example request for creating a broadcast notification might look like:
+
+```bash
+curl -X POST https://[BACKSTAGE_BACKEND]/api/notifications -H "Content-Type: application/json" -H "Authorization: Bearer YOUR_BASE64_SHARED_KEY_TOKEN" -d '{"recipients":{"type":"broadcast"},"payload": {"title": "Title of broadcast message","link": "http://foo.com/bar","severity": "high","topic": "The topic"}}'
+```
+
+## Additional info
```

Comment:
Feedback in general from Adopters is it's a much better experience being able to find the docs all in one place. Ideally we should have all the content here and the `README`s should point to this documentation. Can we make that change, please?

Reply:
Thanks. I will move something relevant upwards (from the READMEs to this file) but not everything.

This document is written the way that the https://github.com/backstage/backstage/pull/24963 will get merged soon. So the basic deployment steps (like  `yarn add` the plugins) are missing here since they will not be relevant.

Considering that, I am rephrasing this section to refer just the implementation and not as a suggestion for the further reading.

Reply:
What you've done here works for me, I don't agree about not having install instructions and I've mentioned that in it's own comment with my thoughts.